extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientBuilderImpl.java

```diff
@@ -386,6 +386,10 @@ public <T> T build(Class<T> aClass) throws IllegalStateException, RestClientDefi
         Integer maxChunkSize = (Integer) getConfiguration().getProperty(QuarkusRestClientProperties.MAX_CHUNK_SIZE);
         if (maxChunkSize != null) {
             clientBuilder.maxChunkSize(maxChunkSize);
+        } else if (restClientsConfig.maxChunkSize.isPresent()) {
+            clientBuilder.maxChunkSize(restClientsConfig.maxChunkSize.get());
+        } else if (restClientsConfig.multipart.maxChunkSize.isPresent()) {
+            clientBuilder.maxChunkSize(restClientsConfig.multipart.maxChunkSize.get());
```

Comment:
Question, not necessarily a change: should it be a cumulative Math.max instead of successive if/then/Else ? I understand there is an order in configuration, then documentation should be clear on that ?

Reply:
Why would be a math max?

The point here is to support the new configuration over the old one, the old one is just a fallback until it's removed completely